client/client.go

```diff
@@ -451,7 +451,7 @@ type OpenConsumerChannelRequest struct {
 }
 
 func (r OpenConsumerChannelRequest) toEstimator(ethClient EthClientGetter) (*bindings.ContractEstimator, error) {
-	return bindings.NewContractEstimator(r.HermesID, bindings.HermesImplementationMetaData.ABI, ethClient.Client())
+	return bindings.NewContractEstimator(r.HermesID, bindings.RegistryMetaData.ABI, ethClient.Client())
```

Comment:
`r.HermesID` -> `r.Registry` (not sure if that is the correct variable name, but registry address should go there)